### PR TITLE
Added Appium iOS capability processor

### PR DESCRIPTION
**Pull request by vania-pooh**

### CODE REVIEW

File: proxy/src/main/java/ru/qatools/gridrouter/caps/AppiumCapabilityProcessor.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    @Override
+    public boolean accept(JsonCapabilities caps) {
+        return caps.getBrowserName().isEmpty();
+    }
+
+    @Override
+    public void process(JsonCapabilities caps) {
+        if (isMac(caps)) {
+            caps.any().put("keepKeyChains", true);
+        }
```
Comment (innokenty) [inline proxy/src/main/java/ru/qatools/gridrouter/caps/AppiumCapabilityProcessor.java]: move the if-check to `accept` maybe?

Comment (vania-pooh) [inline proxy/src/main/java/ru/qatools/gridrouter/caps/AppiumCapabilityProcessor.java]: @innokenty lol. Did it. :)

**Comment by innokenty** [comment]
Also a test would be cool, like this for IE: https://github.com/seleniumkit/gridrouter/blob/master/proxy/src/test/java/ru/qatools/gridrouter/caps/IECapabilityProcessorTest.java

**Comment by vania-pooh** [comment]
One more fix. :)

**Pull request merged**